src/app/organizations/collection/collection.component.html

```diff
@@ -133,11 +133,11 @@ <h5>
                       </span>
                     </div>
                   </div>
-                  <div class="truncate-text-2">
-                    <app-ai-bubble *ngIf="entry.topicSelection === TopicSelectionEnum.AI && !entry.isApprovedAITopic"></app-ai-bubble>
+                  <div>
                     <span class="size-small" [matTooltip]="entry.topic" matTooltipPosition="left">
                       {{ entry.topic }}
                     </span>
+                    <app-ai-bubble *ngIf="entry.topicSelection === TopicSelectionEnum.AI && !entry.isApprovedAITopic"></app-ai-bubble>
```

Comment:
I updated the cards in the collection so they match the search cards more, i.e. the AI bubble is after the topic instead of before.